week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,19 @@
+let list = document.createElement("li");
+let image = document.createElement("img");
+
+let ulList = document.querySelector("#dog");
+ulList.appendChild(list);
+list.appendChild(image);
+let button = document.querySelector("#btn");
+
+button.addEventListener("click", function () {
+  fetch("https://dog.ceo/api/breeds/image/random")
+    .then((response) => response.json())
+    .then((data) => {
+      //console.log(data);
+      image.src = data.message;
+    })
+    .catch((error) => {
+      console.error(error);
```

Comment:
Logging to the console when there's an error is great for us as developers - can you think what else we may want to do in this function to make errors clear to the user on the website too?